api/inventory/v1alpha1/inventory_types.go

```diff
@@ -23,6 +23,35 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+// OAuthConfig defines the configurable attributes that represent the authentication mechanism.  This is currently
+// expected to be a way to acquire a token from an OAuth2 server.
+type OAuthConfig struct {
+	// Url represents the base URL of the authorization server. (e.g., https://keycloak.example.com/realms/oran)
+	Url string `json:"url"`
```

Comment:
Do you need operator-sdk:csv:customresourcedefinitions tags for these fields?

Reply:
fixed